src/prof/src/pprof_types.proto

```diff
@@ -0,0 +1,225 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+// This file was originally copied from
+//
+//    https://github.com/google/pprof/blob/main/proto/profile.proto
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
```suggestion
// Copyright 2016 Google Inc. All Rights Reserved.
// Copyright Materialize, Inc. and contributors. All rights reserved.
//
// This file is derived from the pprof project. The original source code was
// retrieved on September 29, 2023 from:
//
//    https://github.com/google/pprof/blob/f7f687d19a9829062f02a2dd6e9a6fb527dbf143/proto/profile.proto
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```

Reply:
Isn’t line 2 of your suggestion factually wrong? We didn’t change anything in that file, so what would we own the copyright on?

Reply:
The lines that say "This file is derived from ..." and any future modifications to the file.